spec/integ/bridge.spec.js

```diff
@@ -213,16 +222,16 @@ describe("Bridge", function() {
                     room_alias_name: "foo",
                 },
             });
-            bridge.run(101, {}, appService);
+            await bridge.run(101, {}, appService);
+
             await appService.onAliasQuery("#foo:bar");
+
             const room = await bridge.getRoomStore().getMatrixRoom("!abc123:bar");
             expect(room).toBeDefined();
-            if (!room) { done(); return; }
```

Comment:
Removing all of this code where it would allowed the test to pass if things failed.